modules/ROOT/pages/indexes-for-search-performance.adoc

```diff
@@ -247,6 +269,26 @@ The following statement will create a named text index on all relationships with
 CREATE TEXT INDEX rel_text_index_name FOR ()-[r:KNOWS]-() ON (r.interest)
 ----
 
+[discrete]
+[[indexes-create-a-text-index-by-param]]
+===== Create a text index using a parameter
+
+The following statement will create a named text index on all nodes labeled with `Person` and which have the `favoriteColor` string property using a parameter for the index name.
```

Comment:
```suggestion
The following statement will create a named text index on all nodes labeled with `Person` and which have the `favoriteColor` `STRING` property using a parameter for the index name.
```

Reply:
this and the next comment would need to be updated for all text and point examples not just the new ones I believe

Reply:
The formatting will all be fixed in my new chapter